polyglot/piranha/test-resources/go/replacement_cleanup/configurations/rules.toml

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2022 Uber Technologies, Inc.
+#
+# <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy of the License at
+# <p>http://www.apache.org/licenses/LICENSE-2.0
+#
+# <p>Unless required by applicable law or agreed to in writing, software distributed under the
+# License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+[[rules]]
+name = "replace_fmt_Print_with_fmt_Println"
+query = """
+(
+    (call_expression
+        function: (selector_expression
+            operand: (identifier) @func_id
+            field: (field_identifier) @func_field
+        )
+    )
+    (#eq? @func_id "@receiver_name")
+    (#eq? @func_field "@fn_name")
+)
+"""
+replace_node = "func_field"
+replace = "@fn_replacement"
+holes = ["receiver_name", "fn_name", "fn_replacement"]
+
+[[rules]]
+groups = ["newline_cleanup"]
+name = "remove_last_new_line_escape_in_intepreted_string_literal"
+query = """
+(
+    (call_expression
+        arguments: (argument_list
+            (_)*
+            (interpreted_string_literal
+                .
```

Comment:
what is this `.` for?

Reply:
My goal is to use it as an [anchor](https://tree-sitter.github.io/tree-sitter/using-parsers#query-syntax), so it matches the last `interpreted_string_literal` child.